Simplenote/src/androidTest/java/com/automattic/simplenote/utils/EspressoUtils.kt

```diff
@@ -0,0 +1,44 @@
+package com.automattic.simplenote.utils
+
+import android.view.View
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+fun hasTextInputLayoutHintText(expectedErrorText: String): Matcher<View> {
```

Comment:
Question (❓): This function is currently unused, isn't it? Wouldn't it be better to add this when it would be actually needed in order to avoid have unused code in the codebase?

Reply:
Yes, I do not use this function. I will remove it.